vortexasdk/endpoints/voyages_search_enriched_result.py

```diff
@@ -36,3 +36,22 @@ def to_df(self) -> pd.DataFrame:
 
         # converts list to a datafrane
         return pd.DataFrame(data=super().to_list())
+
+
+class VoyagesSearchEnrichedRawResult(Result):
```

Comment:
`search` result - has `to_df()` (converts CSV to a df), doesn't have `to_list()`
`search_raw` result - has `to_list()` (converts a raw JSON response to a list), doesn't have `to_df()`